compute_tools/src/bin/compute_ctl.rs

```diff
@@ -184,6 +184,8 @@ fn main() -> Result<()> {
     let _http_handle =
         launch_http_server(http_port, &compute).expect("cannot launch http endpoint thread");
 
+    let extension_server_port: u16 = http_port;
```

Comment:
I prefer to put it inside compute spec?

Reply:
we are going to try to start loading extensions before the spec arrives